### PR TITLE
Add build instructions

### PR DESCRIPTION
**Pull request by saulhudson**

### CODE REVIEW

**Comment by ishythefishy** [comment]
No way, I was LITERALLY just doing this on a different branch!

File: README.md [hunk 1]
```diff
@@ -31,6 +31,33 @@ Angular Prerender
 Netlify
 ```
 
+### Build Instructions
+Requires Node & NPM.
```
Comment (ishythefishy) [inline README.md:35]: Might be worth specifying that we need Node 14 :)

File: README.md [hunk 2]
```diff
@@ -31,6 +31,33 @@ Angular Prerender
...
+
+1. Ensure Node Package Manager (NPM) and Angular CLI (NG) are installed.
+```
+npm -v
+npm install -g @angular/cli
+ng --version
+```
+2. Fork the reposititory.
```
Comment (ishythefishy) [inline README.md:43]: Spelling error in `reposititory` ;)

File: README.md [hunk 3]
```diff
@@ -31,6 +31,33 @@ Angular Prerender
...
+3. Clone your fork to your local development environment.
+```
+git clone [url]
+```
+4. Change into the Fretonantor directory.
```
Comment (ishythefishy) [inline README.md:48]: Spelling error on `Fretonantor`

File: README.md [hunk 4]
```diff
@@ -31,6 +31,33 @@ Angular Prerender
...
+```
+cd fretonator
+```
+5. Install the dependencies.
+```
+npm install
+```
+6. Run the local development server. This should open automatically in your browser but would be accessible via http://localhost:4200/.
+```
+ng serve --open
```
Comment (ishythefishy) [inline README.md:58]: I never use --open here - is it necessary?

File: README.md (hunk 1 above)
Comment (saulhudson) [reply README.md:35]: Sure I can add, I actually used 12.18.1 which seemed to work on Windows.

**Comment by ishythefishy** [comment]
I have added the NX instructions back in, so we can merge the README instructions together. Thank you so much for this!

File: README.md (hunk 4 above)
Comment (saulhudson) [reply README.md:58]: Just opens it up in a new tab automatically, it's not necessary I just took the command from Angular's website.

Comment (ishythefishy) [reply README.md:58]: Oh nice - maybe an explanation as to what this does? I've learnt something today!

Comment (ishythefishy) [reply README.md:58]: Nice! I've learnt something today!

**Comment by saulhudson** [comment]
No worries, changes pushed, not sure if I need to do anything my end?

**Comment by ishythefishy** [comment]
All merged here - thank you so much! :D

**Comment by saulhudson** [comment]
Wicked no worries glad that worked :)

There's a little of duplication between my merge and yours ... "This project was generated using [Nx]".

**Comment by ishythefishy** [comment]
Yeah I'll sort it - thank you for kicking this off!

**Pull request merged**